### ISSUE TITLE
[GenerateDeviceCodeStub] same enum value

### ISSUE THREAD

**Issue by matsujirushi** [microsoft/vscode-iot-workbench#737]
If enumValue = 0, conflict enum value.
I recommend to set REBUTTON_ACTIONNUM_NOT_CONFIGURED to last line.

## e.g.
**DTDL**
```
      "schema": {
        "@id": "urn:seeedkk:ReButtonInterface:actionNumType:1",
        "@type": "Enum",
        "valueSchema": "integer",
        "enumValues": [
          {
            "name": "SingleClick",
            "enumValue": 0
          },
          {
            "name": "DoubleClick",
            "enumValue": 2
          },
          {
            "name": "TripleClick",
            "enumValue": 3
          },
          {
            "name": "LongPress",
            "enumValue": 10
          },
          {
            "name": "SuperLongPress",
            "enumValue": 11
          }
        ]
      }
```

**Generated code (ReButton_impl.h)**
```
typedef enum REBUTTON_ACTIONNUM_TAG
{
    REBUTTON_ACTIONNUM_NOT_CONFIGURED,
    REBUTTON_ACTIONNUM_SingleClick = 0,
    REBUTTON_ACTIONNUM_DoubleClick = 2,
    REBUTTON_ACTIONNUM_TripleClick = 3,
    REBUTTON_ACTIONNUM_LongPress = 10,
    REBUTTON_ACTIONNUM_SuperLongPress = 11,
    REBUTTON_ACTIONNUM_ERROR
} REBUTTON_ACTIONNUM;
```

REBUTTON_ACTIONNUM_NOT_CONFIGURED and REBUTTON_ACTIONNUM_SingleClick is zero value.

**Comment by dooriya** [comment]
@matsujirushi Thanks for reporting this issue.

I'll remove the **MYENUM_NOT_CONFIGURED** and **MYNUM_ERROR** from the enum definition, instead,  simply add a **MYENUM_UNDEFINED** value to handle the invalid enum value case.

**Comment by dooriya** [comment]
@matsujirushi, this issued have been fixed in the latest CodeGen, you can re-run the **"IoT Plug and Play: Generate Device Code Stub"** command from VS Code, it will automatically upgrade to the latest CodeGen bits.

**Comment by dooriya** [comment]
Fixed in CodeGen 0.6.2 package
